testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/mockprovider/txtimeout/TxTimeoutTestCase.java

```diff
@@ -171,7 +147,7 @@ public void test_negativeTxTimeoutTest() throws Exception {
         try {
             sfsb1.createEmployeeWaitForTxTimeout(false, "Wily", "1 Appletree Lane", 10);
         } catch (Exception e) { // ignore the tx rolled back exception
-            System.out.println("ignoring the " + e.getMessage());
+            //
```

Comment:
yeah and IDE complains if block is empty, you essentially tell tooling that you intentionally left it blank